lib/lightning/invocation.ex

```diff
@@ -252,6 +253,14 @@ defmodule Lightning.Invocation do
     |> Repo.insert()
   end
 
+  def count_invocation_reasons_for_user(%User{id: id}) do
+    from(invocation_reason in Lightning.InvocationReason,
+      where: invocation_reason.user_id == ^id,
+      select: count(invocation_reason.id)
+    )
+    |> Repo.one()
+  end
+
```

Comment:
This looks like it's only being used in tests, please move it there - we don't want to add interfaces that aren't used in the rest of the application.

Reply:
Done. Thanks.